05-policy-changes/policy-example.ipynb

```diff
@@ -0,0 +1,372 @@
+{
```

Comment:
<p>show more example queries</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ContextualAI/examples/pull/25/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
I added one more, but really, it's a better experience exploring queries from the UI then the notebook.

Reply:
Let's link to the UI, you can do that like so:

```python
tenant="" # put the name of your tenant here
print(f"Click on this link to query your Agent: https://app.contextual.ai/{tenant}/agents/{agent_id}/chat")
```